packages/filebrowser/src/listing.ts

```diff
@@ -2307,7 +2307,8 @@ export namespace DirListing {
       narrow.textContent = '...';
       if (!hiddenColumns?.has('is_selected')) {
         const checkboxWrapper = this.createCheckboxWrapperNode({
-          alwaysVisible: true
+          alwaysVisible: true,
+          headerNode:true
```

Comment:
To satisfy the linter:

```suggestion
          headerNode: true
```